etc/picongpu/jureca-jsc/batch_picongpu.profile.example

```diff
@@ -0,0 +1,109 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ######################################### (edit those lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+export proj=$(groups | awk '{print $5}')
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module purge
+module load Intel/2019.0.117-GCC-7.3.0
+module load CMake/3.13.0
+module load IntelMPI/2018.4.274
+module load Python/3.6.6
+module load Boost/1.68.0-Python-3.6.6
+
+# Other Software ##############################################################
+#
+module load zlib/.1.2.11
```

Comment:
typo in version? starts with `.`? (same in `libpng`)

Reply:
No, these are hidden (dunno why) and so accessible only this way.